src/features/counterfactual/ActivateAccount.tsx

```diff
@@ -0,0 +1,186 @@
+      return
+    }
+
+    setTxFlow(undefined)
+  }
+
+  const submitDisabled = !isSubmittable
+
+  return (
+    <TxLayout title="Activate account" hideNonce>
+      <TxCard>
+        <Typography>
+          You&apos;re about to deploy this Safe Account and will have to confirm the transaction with your connected
+          wallet.
+        </Typography>
+
+        <Divider sx={{ mx: -3, my: 2 }} />
+
+        <Grid container spacing={3}>
+          <ReviewRow name="Network" value={<ChainIndicator chainId={chain?.chainId} inline />} />
+          <ReviewRow
+            name="Owners"
+            value={
+              <Box data-testid="review-step-owner-info" className={css.ownersArray}>
+                {undeployedSafe.safeAccountConfig.owners.map((owner, index) => (
+                  <EthHashInfo
+                    address={owner}
+                    shortAddress={false}
+                    showPrefix={false}
+                    showName
+                    hasExplorer
+                    showCopyButton
+                    key={index}
+                  />
+                ))}
+              </Box>
+            }
+          />
+          <ReviewRow
+            name="Threshold"
+            value={
+              <Typography>
+                {undeployedSafe.safeAccountConfig.threshold} out of {undeployedSafe.safeAccountConfig.owners.length}{' '}
+                owner(s)
+              </Typography>
+            }
+          />
+        </Grid>
+
+        <Divider sx={{ mx: -3, my: 2 }} />
+
+        <Grid container spacing={3}>
+          <ReviewRow
+            name="Est. network fee"
+            value={
+              <>
+                <NetworkFee totalFee={totalFee} willRelay={false} chain={chain} />
+
+                <Typography variant="body2" color="text.secondary" mt={1}>
+                  You will have to confirm a transaction with your connected wallet.
+                </Typography>
+              </>
+            }
+          />
+        </Grid>
+
+        {submitError && (
+          <Box mt={1}>
+            <ErrorMessage error={submitError}>Error submitting the transaction. Please try again.</ErrorMessage>
+          </Box>
+        )}
+
+        <Divider sx={{ mx: -3, mt: 2, mb: 1 }} />
+
+        <Box display="flex" flexDirection="row" justifyContent="flex-end" gap={3}>
+          <Button onClick={createSafe} variant="contained" size="stretched" disabled={submitDisabled}>
+            {!isSubmittable ? <CircularProgress size={20} /> : 'Activate'}
+          </Button>
+        </Box>
+      </TxCard>
+    </TxLayout>
+  )
+}
+
+const ActivateAccount = () => {
+  const { safe } = useSafeInfo()
+  const { setTxFlow } = useContext(TxModalContext)
+
+  if (safe.deployed) return null
+
+  const activateAccount = () => {
+    setTxFlow(<ActivateAccountFlow />)
+  }
+
+  return (
+    <Alert severity="info" sx={{ mb: 3 }}>
+      <Typography fontWeight="bold">Want to skip onboarding?</Typography>
```

Comment:
This copy makes little sense. The onboarding has already ended if the user is on this page.
```suggestion
      <Typography fontWeight="bold">Activate your account?</Typography>
```